src/igvfd/types/treatment.py

```diff
@@ -0,0 +1,31 @@
+from snovault import (
+    collection,
+    load_schema,
+    calculated_property
+)
+from .base import (
+    Item
+)
+
+
+@collection(
+    name='treatments',
+    unique_key='uuid',
+    properties={
+        'title': 'Treatment',
+        'description': 'Listing of treatments',
+    })
+class Treatment(Item):
+    item_type = 'treatment'
+    schema = load_schema('igvfd:schemas/treatment.json')
+
+    @calculated_property(
+        schema={
+            'title': 'Title',
+            'type': 'string',
+        }
+    )
+    def title(self, treatment_term_name, amount, amount_units, duration, duration_units):
+        text = 'Treated with {} {} {} for {} {}.'.format(
```

Comment:
I think you should also add a test that makes sure the calculated property is calculated as expected